.rubocop.yml

```diff
@@ -1,5 +1,5 @@
 AllCops:
-  TargetRubyVersion: 2.7
+  TargetRubyVersion: 3.1
```

Comment:
Please no... this isn't a part of supporting rails 7.2 or is it necessary for it, so it should be a separate PR.